File: 단계별로 풀어보기/기본 수학1/1712/손익분기점.cpp
#include <iostream>
using namespace std;

long findBreakEvenPoint(long A, long B, long C){
    if(C<=B)
        return -1;
    else
       return A/(C-B)+1;
}

int main(){
    long A, B, C;
    cin>>A>>B>>C;
    
    cout<<findBreakEvenPoint(A,B,C);
    return 0;
}

File: 단계별로 풀어보기/기본 수학1/1712/손익분기점.py
import sys

def findBreakEvenPoint(A,B,C):
    if(C<=B):
        return -1
    else:
        return A//(C-B)+1
    
    
def init():
    A, B, C = map(int,sys.stdin.readline().split())
    print(findBreakEvenPoint(A,B,C))
    return 0
    
init()
